### ISSUE TITLE
No more development?

### ISSUE THREAD

**Issue by 002gRTEah3mRYH** [beakerbrowser/beaker#1959]
No more development?

**Comment by Kyllingene** [comment]
Already answered in #1944

**Comment by cuibonobo** [comment]
The linked issue is gone...

**Comment by paul90** [comment]
https://github.com/beakerbrowser/beaker/discussions/1944 is a discussion thread, it is still there.

**Comment by cuibonobo** [comment]
Ah, sorry for the noise. The issue was that discussion threads aren't visible for me on the GitHub mobile app for some reason.

**Comment by paul90** [comment]
For those unable to see the discussion thread, I'll quote pfrazee's reply of 10 Sept 2021

> Yeah I'm real sorry to say, I'm going to be closing down this project in the very near future. I'll do a post mortem and share more, but in the intro to my follow-on project I do give a few words on why I felt it was time to change the approach: https://atek.cloud/blog/hello-world

Worth observing that Atek Cloud is also no more, but the blog post mentioned is available via the Internet Archive - https://web.archive.org/web/20220525100741/https://atek.cloud/blog/hello-world

**Comment by pfrazee** [comment]
Yeah I wish I could've given more prompt and definitive answers, but a lot of things take a while (esp around paperwork).

@paul90 has it right, and the missing piece of the puzzle there is that I was recruited as one of the two primary engineers on Bluesky. I couldn't pass up that opportunity; it's values-aligned, well resourced, and the team is really talented.

Beaker is definitely done and I just need to wind it down officially. Atek was the followup project, and it was going to be a difficult product to land but it was a good direction to take P2P technology. The creators of Beaker's p2p tech are now building https://keet.io/ which is also a good direction to go. At some point, I'll get that post-mortem done.

**Comment by milahu** [comment]
> The creators of Beaker's p2p tech are now building [keet.io](https://keet.io/)

looks like https://github.com/berty/berty
